### ISSUE TITLE
RadarChart is missing keyboard accessibility

### ISSUE THREAD

**Issue by saurabhraj123** [recharts/recharts#5390]
## Do you want to request a feature or report a bug?
RadarChart is missing keyboard accessibility. Below is the code for the same.
```javascript
import React, { PureComponent } from "react";
import {
  Radar,
  RadarChart,
  PolarGrid,
  PolarAngleAxis,
  PolarRadiusAxis,
  ResponsiveContainer,
  Tooltip,
} from "recharts";

const data = [
  {
    subject: "Math",
    A: 120,
    B: 110,
    fullMark: 150,
  },
  {
    subject: "Chinese",
    A: 98,
    B: 130,
    fullMark: 150,
  },
  {
    subject: "English",
    A: 86,
    B: 130,
    fullMark: 150,
  },
  {
    subject: "Geography",
    A: 99,
    B: 100,
    fullMark: 150,
  },
  {
    subject: "Physics",
    A: 85,
    B: 90,
    fullMark: 150,
  },
  {
    subject: "History",
    A: 65,
    B: 85,
    fullMark: 150,
  },
];

export default class Example extends PureComponent {
  static demoUrl = "https://codesandbox.io/p/sandbox/simple-radar-chart-2p5sxm";

  render() {
    return (
      <ResponsiveContainer width="50%" height="50%">
        <RadarChart
          cx="50%"
          cy="50%"
          outerRadius="80%"
          data={data}
          accessibilityLayer={true}
        >
          <PolarGrid />
          <PolarAngleAxis dataKey="subject" />
          <PolarRadiusAxis />
          <Radar
            name="Mike"
            dataKey="A"
            stroke="#8884d8"
            fill="#8884d8"
            fillOpacity={0.6}
          />
          <Tooltip />
        </RadarChart>
      </ResponsiveContainer>
    );
  }
}
``` 

Please help me with this. Thanks!

## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?
**2.10.4**

**Comment by ckifer** [comment]
Yep this is known. We'll probably get 3.0 out and then work on keyboard controls for polar charts

**Comment by aallred406** [comment]
> Yep this is known. We'll probably get 3.0 out and then work on keyboard controls for polar charts

@ckifer PieChart is also missing keyboard navigation. Each segment does receive focus but the tooltip does not appear on focus. Any chance that would also be included in a 3.0 release?

**Comment by ckifer** [comment]
Yep same deal with all the Polar charts

**Comment by ckifer** [comment]
Since this project is maintained on spare time there isn't a release date planned but hopefully in the next couple of months. Trying to test while squeezing a few features into the alpha version
